sklearn/decomposition/nmf.py

```diff
@@ -1021,9 +1021,6 @@ def fit_transform(self, X, y=None, W=None, H=None):
         components_ : array-like, shape (n_components, n_features)
             Factorization matrix, sometimes called 'dictionary'.
 
-        n_iter_ : int
```

Comment:
I was commenting that there should be no Attributes section *at all* in a method docstring. it belongs in the class docstring.